exec/java-exec/src/main/java/org/apache/drill/exec/expr/stat/RangeExprEvaluator.java

```diff
@@ -259,17 +259,26 @@ public long getRowCount() {
       final ValueHolder maxFuncHolder = InterpreterEvaluator.evaluateFunction(interpreter, args2, holderExpr.getName());
 
       switch (destType) {
-      //TODO : need handle # of nulls.
       case INT:
-        return getStatistics( ((IntHolder)minFuncHolder).value, ((IntHolder)maxFuncHolder).value);
+        Statistics si = getStatistics( ((IntHolder)minFuncHolder).value, ((IntHolder)maxFuncHolder).value);
+        si.setNumNulls(input.getNumNulls());
```

Comment:
move `si.setNumNulls()` outside of the `switch` block.

Reply:
Possible? Can you explain?

Reply:
Before switch: `Statistics<?> statistics;`;
in each case except for `default`: `statistics = getStatistics(...);`
after switch `statistics.setNumNulls(); return statistics;`

Reply:
Done